### ISSUE TITLE
Republish with Legends of Learning support

### ISSUE THREAD

**Issue by samreid** [phetsims/energy-skate-park-basics#375]
@kathy-phet said:

> Some good ones for legends of learning priorities would be force and motion: basics and energy skate park : basics ... assuming they had these on their list too. 

I'll check for the list.

**Comment by samreid** [comment]
This list https://github.com/phetsims/special-ops/issues/70#issuecomment-305887215 includes Energy Skate Park: Basics, so I think we are a go.  @amanda-phet we will need icons for this sim.

**Comment by samreid** [comment]
Next step will be to test master and see if it works in the LOL harness with respect to pausing while dragging.

**Comment by samreid** [comment]
Testing master in the harness with ?legendsOfLearning works great, no dragging problems for skater, track or track vertices with "pause in 5 seconds".  Skater motion pauses correctly.  This is ready for a RC.

**Comment by samreid** [comment]
Pending testing in the above issue, this one is going on hold.

**Comment by samreid** [comment]
Well, I guess the need for icons is not on-hold, I'll reassign this to @amanda-phet.  Please reassign to me once icons are ready.

**Comment by amanda-phet** [comment]
@samreid Here is the google drive folder with icons: https://drive.google.com/drive/folders/0B3v_ZP9Kq36Ca19aZjQ2WnE0bDA

**Comment by samreid** [comment]
Thanks @amanda-phet, looks great.  On hold until testing completes.

**Comment by samreid** [comment]
I need to include phetsims/joist@63046a6 before RC testing continues.

**Comment by samreid** [comment]
Please see also https://github.com/phetsims/joist/issues/449#issuecomment-340263119, we are considering picking up some other commits for this maintenance release.

**Comment by samreid** [comment]
It seems we need the following cherry picks:
https://github.com/phetsims/joist/issues/449#issuecomment-340573495 (for screen names)
https://github.com/phetsims/joist/commit/5346db7405f5dd08dfc82a0a5041e8364c719eda (for correct pause behavior)

**Comment by samreid** [comment]
Based on the number of issues raised during the RC test, it is unclear whether we will be using the same SHAs or not for next RC.

**Comment by samreid** [comment]
On hold until we address the issues raised during the RC, see https://github.com/phetsims/QA/issues/50

**Comment by samreid** [comment]
It will also be good to publish this since it fixes https://github.com/phetsims/energy-skate-park-basics/issues/127

**Comment by samreid** [comment]
On hold until we discuss https://github.com/phetsims/QA/issues/50

**Comment by samreid** [comment]
On Thursday's meeting, we decided to reprioritize getting this ready for republication for PhET brand (with Legends of Learning support), so that we can republish before @jessegreenberg makes a large-scale refactoring.  The last RC test was completed here: https://github.com/phetsims/QA/issues/50 which included full testing for PhET-iO issues.  I recommend we defer PhET-iO work on this sim until we can have a stable PhET-iO design meeting for it (but not sure whether this was part of the plan).  Skimming through open issues in https://github.com/phetsims/QA/issues/50 it seems most open issues were surrounding PhET-iO.

I skimmed through https://github.com/phetsims/energy-skate-park-basics/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3Adev%3Aphet-io+-label%3Apriority%3A5-deferred+-label%3A%22sim%3AFor+1.1%2B%22+-label%3Adev%3Acode-review (note I suppressed several labels) and the main issue that stood out to me for fixing before next RC was:
https://github.com/phetsims/energy-skate-park-basics/issues/413

But it would be great if @jessegreenberg could also look through the open issues and see if anything else should be done before next RC.  As decided on Thursday, @jessegreenberg will take the lead on this and consult with me as needed.

**Comment by jessegreenberg** [comment]
Thanks @samreid, after reviewing, the only thing that stood out to me was this potential memory leak:
https://github.com/phetsims/energy-skate-park-basics/issues/399

Seems like the rest can be addressed as part of work in https://github.com/phetsims/energy-skate-park.

**Comment by jessegreenberg** [comment]
This issue has also resurfaced: https://github.com/phetsims/energy-skate-park-basics/issues/205

**Comment by jessegreenberg** [comment]
#205, #399, and #413 are wrapped up, so this sim is ready for republication with PhET brand. https://github.com/phetsims/energy-skate-park-basics/issues/411 will determine if a PhET-iO version needs  to be done at the same time. Once that issue is clarified we can create an RC.

**Comment by jessegreenberg** [comment]
This sim is currently in dev testing in https://github.com/phetsims/QA/issues/134. We are also publishing a new PhET-iO version out of master in #411.

**Comment by arouinfar** [comment]
The 1.4 version of ESPB was never published with LoL support. We are currently serving version 1.1.28 on the website, and we'll pick up LoL support the next time this sim is republished. 

Republishing a simulation solely for LoL support is not a priority, so I'm going to close this issue.
